datahub-web-react/src/app/ingest/source/builder/RecipeForm/FormField.tsx

```diff
@@ -80,6 +84,30 @@ function ListField({ field, removeMargin }: ListFieldProps) {
     );
 }
 
+function SelectField({ field }: SelectFieldProps) {
```

Comment:
could you make it so that the select dropdown and the label are on the same line? instead of the selector being on a second line right now. And can you also make the selector not quite so wide? so like `width: 80%;`

Reply:
Checked and fixed.